docusaurus/docs/iOS/05-ui-cookbook/16-snapshot.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: Snapshot
```

Comment:
Just Snapshot is a bit generic, maybe "Video Feed Snapshot" or "Capturing Snapshots"?